weave-js/src/components/Panel2/PanelRootBrowser/ProjectObjectsTable.tsx

```diff
@@ -130,3 +131,22 @@ export const ProjectObjectsTable: React.FC<
     </PanelContextProvider>
   );
 };
+
+const EmptyStateComp: FC = () => {
```

Comment:
Any benefit to declaring this FC?

Reply:
Nope, same as above, I thought we were gonna do something more complicated. I've changed it to inlining `EmptyStateContainer` instead.